### PR TITLE
ENH: Surface and root workflow

### PR DESCRIPTION
**Pull request by mgxd**

### CODE REVIEW

File: nibabies/cli/run.py [hunk 1]
```diff
@@ -84,6 +84,12 @@ def get_parser():
         default=False,
         help="Use low-quality tools for speed - TESTING ONLY",
     )
+    parser.add_argument(
+        "--age",
```
Comment (oesteban) [inline nibabies/cli/run.py:88]: shouldn't this be extracted from the BIDS structure?

File: nibabies/workflows/base.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+def init_infant_anat_wf(
+    *,
+    in_template,
+    template_specs,
+    age_months,
+    mri_scheme,
+    omp_nthreads,
+    output_dir
+):
+    inputnode = pe.Node(niu.IdentityInterface(fields=["in_files"], ["in_seg"]), name='inputnode')
```
Comment (oesteban) [inline nibabies/workflows/base.py:17]:
```suggestion
    inputnode = pe.Node(niu.IdentityInterface(fields=("in_files", "in_seg")), name='inputnode')
```

File: nibabies/workflows/surfaces.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    # the brainmask applied, and then feed that into baby freesurfer
+    # down the line, we might want to inject another segmentation to replace baby FS's aseg.
+    recon = pe.Node(InfantReconAll(age=age_months), name='reconall')
+    wf.connect([
+        (inputnode, recon, [('t1_file', 't1_file'),
+                            ('masked_file', 'mask_file')]),
+    ])
+
+    # convert generated surfaces to GIFTIs
+    gifti_surface_wf = init_gifti_surface_wf(
```
Comment (oesteban) [inline nibabies/workflows/surfaces.py:30]: I guess the definition of this should be somewhere right? Even if it is a stub

File: nibabies/cli/run.py (hunk 1 above)
Comment (mgxd) [reply nibabies/cli/run.py:88]: our test cases are currently not in BIDS structure 😄 

but down the line, yes if possible - `participants.tsv` is not mandatory.

**Pull request merged**